common/models/create_environment_request.go

```diff
@@ -22,23 +21,46 @@ func (r CreateEnvironmentRequest) Validate() error {
 		return fmt.Errorf("EnvironmentName is required")
 	}
 
-	if r.Scale < 0 {
-		return fmt.Errorf("Scale must be a positive integer")
+	switch r.EnvironmentType {
+	case EnvironmentTypeStatic, EnvironmentTypeDynamic:
```

Comment:
Should this return anything or have an info line?

Reply:
I don't think there is an need to do anything on success cases here